projects/packages/changelogger/changelog/update-php-requirements

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Dropping support for an old version should be a major bump, as it's a BC break for anyone still using the old version.

I'm tempted to suggest leaving changelogger as-is for the moment though. Bumping this should also involve checking whether any of the other dependencies can be updated (e.g. looks like wikimedia/at-ease ^2.0 will cover PHP 7+) and removing `projects/packages/changelogger/src/php5/`.

Reply:
👍 I switched to a major version bump in 6caa55c72efd1f4cd8352fb39b30055590d6efbc